docs/guides/digital-assets/transfer-lsp7-digital-asset.md

```diff
@@ -173,6 +173,8 @@ Finally we send the transaction and transfer the tokens from a Universal Profile
   
   <TabItem value="web3js" label="web3.js">
 
+<!-- prettier-ignore-start -->
```

Comment:
Why not prettifying the JS code blocks as well?
It would be good to ensure they are correctly formatted for the developer guides